ui/src/shared/components/AutoRefresh.js

```diff
@@ -150,3 +148,5 @@ export default function AutoRefresh(ComposedComponent) {
 
   return wrapper
 }
+
+export default AutoRefresh
```

Comment:
What's the argument for this pattern over the following? Should we add this type of thing to our central style guide so we can converge on a style?

```
export const AutoRefresh = (ComposedComponent) => {
```